pkg/destroy/gcp/cloudcontroller.go

```diff
@@ -26,17 +26,7 @@ func (o *ClusterUninstaller) listCloudControllerBackendServices(ctx context.Cont
 		urls.Insert(instanceGroup.url)
 	}
 	filter := "name eq \"a[0-9a-f]{30,50}\""
-	return o.listBackendServicesWithFilter(ctx, "items(name,backends),nextPageToken", filter, func(item *compute.BackendService) bool {
-		if len(item.Backends) == 0 {
-			return false
-		}
-		for _, backend := range item.Backends {
-			if !urls.Has(backend.Group) {
-				return false
-			}
-		}
-		return true
-	}, gcpRegionalResource)
+	return o.listBackendServicesWithFilter(ctx, regionBackendServiceResource, "items(name,backends),nextPageToken", filter)
```

Comment:
`urls` is not being used anymore. Is that on purpose? If so, we can remove `urls`.